packages/api/src/controllers/region.js

```diff
@@ -0,0 +1,57 @@
+import Router from 'express/lib/router'
+import { authMiddleware } from '../middleware'
+
+const app = Router()
+
+const getOrchestrators = async (req, res, next) => {
+  const orchestrators = await req.getOrchestrators(req)
+
+  return res.json(orchestrators.map(({ address }) => ({ address })))
+}
+
+app.get('/:region', authMiddleware({ admin: true }), async (req, res, next) => {
+  const region = await req.store.get(`region/${req.params.region}`)
+  return res.json(region)
+})
+
+app.post(
+  '/:region',
+  authMiddleware({ admin: true }),
+  async (req, res, next) => {
+    if (!req.body || !req.body.region) {
+      res.status(406)
+      return res.json({ error: 'region field is required' })
+    }
+
+    if (!req.body.orchestrators || !req.body.orchestrators.length) {
```

Comment:
If it's omitted, that can be an error. But I think an empty array is valid; there might not be an orchestrator in a given region.